Entities/Items/Boulder/Boulder.as

```diff
@@ -239,6 +268,13 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	return damage;
 }
 
+bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
+{
+	string name = blob.getName();
+	if ((name == "archer" || name == "knight" || name == "builder") && this.get_u8("launch team") == blob.getTeamNum()) return false;
```

Comment:
How about just make it not collide with any player blobs? This would work better with mods that have custom classes
```suggestion
	if (blob.getPlayer() !is null && this.get_u8("launch team") == blob.getTeamNum()) return false;
```